azure-devops/projects/cgn-onboarding-portal-projects/secrets.tf

```
@@ -12,6 +12,10 @@ variable "secrets" {
     "io-azure-devops-github-USERNAME",
     "pagopa-npm-bot-TOKEN",
     "TTDIO-SPN-TENANTID",
+    "TTDIO-UAT-GCNPORTAL-SUBSCRIPTION-ID",
+    "TTDIO-PROD-GCNPORTAL-SUBSCRIPTION-ID",
+    "cgnportal-sonarqube-URL",
+    "cgnportal-sonarqube-TOKEN"
```

```suggestion
    "cgnportal-sonarqube-URL",
    "cgnportal-sonarqube-TOKEN"
```
Is it your sonarqube or sonarcloud?

---

Should be this one https://github.com/pagopa/cgn-onboarding-portal-infra/blob/master/code/sonarqube.tf#L39

cgnportal-sonarqube-URL should be "https://cgnonboardingportal-u-sonarqube.westeurope.azurecontainer.io/"

It's not yet deployed in UAT due to enable_sonarqube being false